ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
```

Comment:
Is it possible to avoid `Windows.Storage`. I'm thinking that by not taking a Windows dependency and instead just using generic libraries, we can ultimately implement this on other platforms if we wished.

If we can't, it's not a big deal as I don't foresee us porting this any time soon.